controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/JobRunnerTest.java

```diff
@@ -337,6 +337,25 @@ public void historyPruning() {
         assertFalse(jobs.firstFailing(jobId).isPresent());
     }
 
+    @Test
+    public void onlySuccessfulRunExpiresThenAnotherFails() {
+        DeploymentTester tester = new DeploymentTester();
+        JobController jobs = tester.controller().jobController();
```

Comment:
To var or not, that is the question.

Reply:
Yeah, I'm not settled on that one yet :)